src/smbprotocol/session.py

```diff
@@ -272,7 +271,7 @@ def connect(self):
                 protocol=self.auth_protocol,
             )
         except spnego.exceptions.SpnegoError as err:
-            raise SMBAuthenticationError("Failed to authenticate with server: %s" % str(err.message))
+            raise SMBAuthenticationError("Failed to authenticate with server: %s", err.message)
```

Comment:
I think you still want `%` here or to convert it to an f string. Unless exceptions support such a thing?

Reply:
Replaced with `raise SMBAuthenticationError(msg) from err`